pkgs/applications/video/kodi/addons/svtplay/default.nix

```diff
@@ -2,13 +2,13 @@
 buildKodiAddon rec {
   pname = "svtplay";
   namespace = "plugin.video.svtplay";
-  version = "5.1.12";
+  version = "5.1.16";
```

Comment:
seems like version 5.1.16 is meant for kodi 20. It does not work with kodi 19 as in this PR. Version 5.1.15 does work with kodi 19 for me.